Kwenta/kwenta
# Disable the place order button when market cap reached

## Description
When a market hits its max Long/Short OI cap, the "Open Position/Modify Position" button will be disabled and stated "Open Interest cap reached" until the OI reaches levels below the max cap. 

## Related issue
#673 

## Motivation and Context
Although the trader is not able to open the new position if the Open Interest cap has been reached, they can click on the `Open Position`button. Try to improve the UX in this case.

## How Has This Been Tested?
1. Find a market that Long OI cap has been reached
2. See the `open position` button show the message `Open Interest cap reached`
3. Find a market that Short OI cap hasn't been reached
4. See the `open position` button show the normal info

## Screenshots (if appropriate):
Note that I manually set both Long and Short OI cap as $900K. 
Long OI has been reached ($995K)
![image](https://user-images.githubusercontent.com/4819006/165831797-e140f097-2aeb-4344-a64d-c2352ba65878.png)
Short OI hasn't been reached ($0.05K)
![image](https://user-images.githubusercontent.com/4819006/165831860-e934b1e3-0719-4187-83e5-39c717b282b5.png)
